src/sentry_envelope.c

```diff
@@ -177,7 +96,7 @@ envelope_item_get_category(const sentry_envelope_item_t *item)
     } else if (sentry__string_eq(ty, "transaction")) {
         return SENTRY_RL_CATEGORY_TRANSACTION;
     }
-    // NOTE: the `type` here can be `event`, `minidump` or `attachment`.
+    // NOTE: the `type` here can be `event` or `attachment`.
     // Ideally, attachments should have their own RL_CATEGORY.
```

Comment:
Note for later: This comment is true, but the logic is complicated and not fully specified yet. It's good if it stays like this for now.